src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/TensorsInfo.cs

```diff
@@ -380,7 +499,45 @@ public void SetDimension(int[] dimension)
                 Dimension = (int[])dimension.Clone();
             }
 
-            public string Name { get; set; } = null;
+            public int GetSize()
```

Comment:
How about use property?

```suggestion
            public int Size { get; }
```

Reply:
ACK!